examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -53,6 +53,16 @@
 
 /**
  * End-to-end tests of WordCount.
+ *
+ * <p>Input text document is available from the following sources:
```

Comment:
@lukecwik. I tried with a txt file in resource dir and running WordCountIT locally(with Flink runner). As far as I investigate, IOChannelFactory failed the reading, since it can't validate the file which is inside a jar. I'm not very knowledgeable on how maven packaging and executing IT test. Maybe there are someways to do the validation check without absolute path. Any suggestions?

However, It's a good idea to package a small set of static data for example pipelines and tests. That will simplify step and complicity of running examples for beginner.